### ISSUE TITLE
kube-apiserver.yaml image tag in documentation refers to old version; is the only available version

### ISSUE THREAD

**Issue by jdfless** [kubernetes/kubernetes#19738]
*TL;DR - Add more release versions to gcr.io/google_containers/\<package\> and tag by version*

Currently, the kube-apiserver.yaml linked to in the in the 1.1 version of the docs for High Availability (http://kubernetes.io/v1.1/docs/admin/high-availability.html) refers to:
`image: gcr.io/google_containers/kube-apiserver:9680e782e08a1a1c94c656190011bd02`

This image tag has kube-apiserver version 0.18.0. When I tried to get version 1.1.3 (the version we are running / looking to replace with a containerized version) I realized this was the only image that exists.
```
~$ curl -k -s -X GET https://gcr.io/v2/google_containers/kube-apiserver/tags/list | jq -r '.tags[]'
9680e782e08a1a1c94c656190011bd02
```

The main issue here is that there is only a single version available for the apiserver. This is also the case for the kube-controller-manager, kube-scheduler, and podmaster.

There are multiple versions of etcd but there isn't a standardized pattern as 2.0.3 is preceded by a 'v' and the rest are not: 
```    
~$ curl -k -s -X GET https://gcr.io/v2/google_containers/etcd/tags/list | jq -r '.tags[]'
2.0.12
2.0.13
2.0.8
2.0.9
2.0.9.1
2.1.0-alpha.0
2.2.1
v2.0.3
```
A further improvement would be to tag each image with just a version number.

**Comment by jdfless** [comment]
As an addendum - I found the hyperkube image which is nicely tagged - but the above images should still either be deprecated or brought in line. Also, the docs still need to be upgraded. This issue may duplicate https://github.com/kubernetes/kubernetes/issues/11751.

**Comment by luxas** [comment]
This has been fixed in #19061, but it will show up first in `v1.2.0` releases unfortunately
